bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerV2Handler.java

```diff
@@ -322,12 +325,26 @@ public void initialize() {
     }
 
     private String getReadUrl() {
```

Comment:
As this change adds a `IllegalArgumentException` the method should be annotated with this exception so callers are forced to handle this case.

Reply:
'Leave it for now. I will come up with a follow up PR to refactor some things. Hopefully you are available to verify?

Reply:
Sure! I'm out of town this week and also on vacation starting in 3 but I'll try to fit it in somewhere.